modal/modal.css

```diff
@@ -0,0 +1,49 @@
+@import url("../styles.css");
+
+.header-btn {
+  background-color: var(--primary-dark);
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 3px 1px -2px #0003, 0 2px 2px #00000024, 0 1px 5px #0000001f;
+  color: var(--font);
+  font-size: 0.9rem;
+  padding: 10px 30px;
+  cursor: pointer;
+  margin-bottom: 30px;
+}
+.header-modal {
+  border-radius: 5px;
+  box-shadow: #183040 1.95px 1.95px 2.6px;
+  margin: auto;
+  visibility: hidden;
+  width: 21vw;
+}
+
+.header-title {
+  font-size: 1.2rem;
+  font-weight: 500;
+  padding: 1rem 1rem;
+}
+
+.header-subtitle {
+  font-size: 1rem;
+  padding: 0rem 1rem;
+}
+
+.button-container{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  padding: 1rem 1rem;
+}
```

Comment:
Shift your button to the end of the container and also reduce the size of the button. It's looking big according to the container. By default its flex-direction row. Use justify-content:flex-end here and give some gap

Reply:
Thank you! done. 😃